src/SearchInput.tsx

```diff
@@ -26,31 +26,17 @@ export const SearchInput = ({
   ...rest
 }: Props) => {
   return (
-    <form onSubmit={onSubmit}>
-      <div className="searchInput">
-        <input
-          className="searchInput__input"
-          type="text"
-          autoComplete="off"
-          autoCapitalize="off"
-          spellCheck="false"
-          autoFocus={autoFocus}
-          placeholder={placeholder}
-          value={value}
-          onChange={onChange}
-          {...rest}
-        />
-
-        {isLoading ? (
-          <div className="searchInput__submitButton">
-            <VuiSpinner size="xs" />
-          </div>
-        ) : (
-          <button className="searchInput__submitButton" onClick={onSubmit}>
-            <BiSearch size="20px" />
-          </button>
-        )}
-      </div>
-    </form>
+    <input
+      className="searchInput__input"
```

Comment:
Nit: I'd rename this `searchInput`

Reply:
https://github.com/vectara/react-search/pull/5/commits/c6fd2694b8441ae3566af22dd67d8b51f182aae7